stage1/stop/stop_fly.mk

```diff
@@ -0,0 +1 @@
+include stage1_fly/makelib/aci_binary.mk
```

Comment:
It would probably be cleaner if you put the code for the fly stage1 image in stage1_fly directory. It could be even simpler, since for fly stage1 image there are no child processes at all.